lib/floki.ex

```diff
@@ -152,13 +168,20 @@ defmodule Floki do
 
   ## Options
 
+    * `:attributes_as_maps` - Change the behaviour of the parser to return the attributes
+      as maps, instead of a list of `{"key", "value"}`. Remember that maps are no longer 
+      ordered since OTP 26. Default to `false`.
+
     * `:html_parser` - The module of the backend that is responsible for parsing
       the HTML string. By default it is set to the built-in parser, and the module
       name is equal to `Floki.HTMLParser.Mochiweb`, or from the value of the
       application env of the same name.
 
       See https://github.com/philss/floki#alternative-html-parsers for more details.
 
+    * `:parser_args` - A list of options to the parser. This can be used to pass options
+      that are specific for a given parser. Defaults to an empty list.
```

Comment:
I am curious, what would be the use case for this option?

Reply:
This was introduced to change the behaviour of a specific parser. Like for the `FastHTML`, that we can pass a `:context` option to it - we do that here: https://github.com/philss/floki/blob/2d5c282099764f12a21ea37d644b9ea09971d0eb/test/floki_test.exs#L242.

It was undocumented before, so I added a mention about it.

Reply:
Perfect!